packages/react-graphql-universal-provider/src/test/GraphQLUniversalProvider.test.tsx

```diff
@@ -1,31 +1,42 @@
 import React from 'react';
-import {ApolloClient} from 'apollo-client';
 import {InMemoryCache} from 'apollo-cache-inmemory';
 import {ApolloLink} from 'apollo-link';
 import {extract} from '@shopify/react-effect/server';
 import {mount} from '@shopify/react-testing';
 import {HtmlManager, HtmlContext} from '@shopify/react-html';
-import {ApolloProvider, SsrExtractableLink} from '@shopify/react-graphql';
+import {ApolloProvider} from '@shopify/react-graphql';
 
 import {GraphQLUniversalProvider} from '../GraphQLUniversalProvider';
 
-jest.mock('@shopify/react-graphql', () => {
-  /* eslint-disable @typescript-eslint/no-var-requires */
-  const ApolloClient = require('apollo-client');
-  const InMemoryCache = require('apollo-cache-inmemory');
-  const ApolloLink = require('apollo-link');
-  /* eslint-enable @typescript-eslint/no-var-requires */
+jest.mock('apollo-client', () => {
```

Comment:
Took me a bit of time to figure out he mock situation, but I am happy with the result.

This implementation mock ApolloClient while calling the actual for result.
It is important to note that `apollo-client` uses alias for both default and name export for ApolloClient. I mocked both to be more maintainable.